src/CalcViewModel/Common/Utils.cpp

```diff
@@ -86,11 +86,11 @@ bool Utils::IsLastCharacterTarget(_In_ wstring const &input, _In_ wchar_t target
     return !input.empty() && input.back() == target;
 }
 
-//return wstring after removing characters like space, comma, and double quotes
+//return wstring after removing characters like space, comma, double quotes, and monetary prefix currency symbols supported by the Windows keyboard
 wstring Utils::RemoveUnwantedCharsFromWstring(wstring input)
 {
-    wchar_t unWantedChars[] = { L' ', L',', L'"', 8234, 8235, 8236, 8237 };
-    return RemoveUnwantedCharsFromWstring(input, unWantedChars, 6);
+    wchar_t unWantedChars[] = { L' ', L',', L'"', 165, 164, 8373, 36, 8353, 8361, 8362, 8358, 8377, 163, 8364, 8234, 8235, 8236, 8237 };
```

Comment:
This file supports unicode characters, meaning you can embed the character you want to ignore, ie `L'€'`.  Please update the values here so the array is readable.

Reply:
When I embed the character `L'€'` into the array, during runtime it becomes `â`.

Reply:
If I had to guess I'd say that your output does not support unicode or is using a different encoding

Reply:
@willibrandon, thanks for trying my suggestion.  When I try locally, I get a warning.  Let's ignore my suggestion and fall back to using unicode literals, ie `L'\u20AC'` is the Euro symbol.  Then please add a comment for each unicode symbol.  What I'm trying to get out of this is that the array should be readable so that other developers don't need to go look up each symbol.

Reply:
@danbelcher-MSFT , may I ask what warning you are receiving? The reason I ask is because I'm having the same problem with writing unit tests to validate the behavior. When I write the unit tests using unicode characters, some of the currency symbols get messed up in the same way. File encoding is set to Unicode (UTF-8 without signature) - Codepage 65001 so I'm not entirely sure why this is happening. One thing I have noticed is if I use the /utf-8 option for the CalculatorUnitTests project [as specified here](https://docs.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-and-executable-character-sets-to-utf-8) then the unicode characters do not get messed up.

This is not a topic I'm very experienced in so any help would be very appreciated.

Reply:
@danbelcher-MSFT , it looks like a fix for this is already in the works #295.

Reply:
Yep, the PR for that was merged, so you can merge from master and see if you can get the embedded Euro characters etc working again.

Reply:
Sorry, I confused the nature of that PR.  We still need to add the /utf8 compiler flag, so this won't work yet.

Reply:
Hey @willibrandon, I don't want to block your PR waiting for us to figure out encoding problems.  What I care most about is readability of the code.  I'm fine with what you had for your first iteration (the raw numerical value of the monetary characters), but please add comments so that other readers can know what the values represent.  Then we can get your change checked-in.

Reply:
@danbelcher-MSFT , I have added comments and made the requested changes. Please let me know if anything else needs to be done.